package.json

```diff
@@ -1,6 +1,6 @@
 {
   "engines": {
-    "node": "12.18.3"
+    "node": "12.x"
```

Comment:
we just need to be aware of docker alpine package limitation

Reply:
if its not using v12 its gonna blow up